docs/packages/pkg/libusb.rst

```diff
@@ -0,0 +1,29 @@
+.. spelling::
+
+    libusb
+
+.. index::
+  single: unsorted ; libusb
+
+.. _pkg.libusb:
+
+libusb
+======
+
+-  `Official <https://libusb.info>`__
+-  `Hunterized <https://github.com/libusb/libusb>`__
+-  `Example <https://github.com/ruslo/hunter/blob/master/examples/libusb/CMakeLists.txt>`__
+-  Added by `Sebastien Collier <https://github.com/sjdrc>`__ (`pr-1830 <https://github.com/ruslo/hunter/pull/1830>`__)
+
+.. literalinclude:: /../examples/libusb/CMakeLists.txt
+  :language: cmake
+  :start-after: # DOCUMENTATION_START {
+  :end-before: # DOCUMENTATION_END }
+
+Pitfalls
+--------
+
+- Requirements for Ubuntu:
+
+.. code::
```

Comment:
Use:
```
.. code:: none
```

- https://travis-ci.org/ruslo/hunter/builds/523035503#L2032

Reply:
The failure to build was actually due to needing a blank line after `.. code::` and the `apt-get` line

Reply:
Ok, I think I confused it with `code-block`.